tools/wptserve/tests/functional/test_request.py

```diff
@@ -58,15 +58,11 @@ def handler(request, response):
 
         try:
             resp = self.request(route[1], method="POST", body="1"*20)
-        except HTTPError:
-            # Cleanup and fail test
+            self.assertEqual(200, resp.getcode())
```

Comment:
Move the max_buffer_size change to right before the `try` so that execptions in other places don't result in it not being reset.